IdentityServer/v7/docs/content/diagnostics/otel/metrics.md

```diff
@@ -7,66 +7,74 @@ weight: 10
 (added in v7.0)
 
 The metric counters are designed to provide a high level overview. They are designed to not contain any
-sensitive information. The counters may contain tags to indicate the source of the data.
+sensitive information. The counters may contain tags to indicate the source of the data. Metrics are
+intended to be used to provide overall health metrics and are typically used to show graphs on
+a dashboard or to set up monitoring rules. If the metrics indicates that something has happened,
+the traces and logs are used to find out the reason. Open Telemetry monitoring tools might have
+provide features to find the traces and logs corresponding to certain metrics.
+
+IdentityServer emits metrics from the IdentityServer library. Our quick start for the UI also
+[contains metrics](#metrics-in-the-ui) that can be used as a starting point for monitoring UI events.
 
 ## High level Metrics
 These metrics are instrumented by the IdentityServer middleware and services and are
 intended to describe the overall usage and health of the system. They could provide the
-starting point for building a metrics dashboard.
+starting point for building a metrics dashboard. The high level metrics are enabled through
+the `Duende.IdentityServer.Telemetry.ServiceName` meter.
 
-#### Telemetry.Metrics.Counters.Failure/Success
-Counter names: *failure*/*success*
+#### Telemetry.Metrics.Counters.Operation
+Counter name: *tokenservice.operation*
 
-Aggregated counter of failed and successful operations. Note that it is expected to have failures during normal
-operation of IdentityServer. The failure/success ratio can be used as a very high level health metric.
+Aggregated counter of failed and successful operations. Note that it is expected to have some failures during
+normal operation of IdentityServer. There should be no internal_errors though, they indicate that there are
+unhandled exceptions. The failure/success ratio can be used as a very high level health metric.
 |Tag|Description|
 |---|---|
 |error | Error label on errors|
+|result | Success, error or internal_error |
 |client | Id of client requesting the operation. May be empty. |
 
 #### Telemetry.Metrics.Counters.ActiveRequests
 Counter name: *active_requests*
 
 Gauge/up-down counter that shows current active requests that are processed by any IdentityServer endpoint. 
 Note that the pages in the user interface are not IdentityServer endpoints and are not included in this count.
+This metrics
 |Tag|Description|
 |---|---|
 |endpoint | The type name for the endpoint processor |
 |path | The path of the request |
 
-#### Telemetry.Metrics.Counters.UnHandledException
-Counter name: *unhandled_exception*
- 
-Number of globally unhandled exceptions from IdentityServer.
-|Tag|Description|
-|---|---|
-|type | The type name of the exception |
-|method | The name of the method where the exception was thrown. |
+### Detailed Metrics - Experimental
```

Comment:
Our style sheet makes h3 and h4 indistinguishable. I try to avoid h3s. (We probably should change the style sheet, come to think of it, but that's a bigger change than we're doing here obviously).